tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -843,7 +844,7 @@ FileContent SwiftASTManager::Implementation::getFileContent(
     StringRef UnresolvedPath, bool IsPrimary,
     llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> FileSystem,
     std::string &Error) const {
-  std::string FilePath = SwiftLangSupport::resolvePathSymlinks(UnresolvedPath);
+  auto FilePath = resolveSymbolicLinks(UnresolvedPath);
```

Comment:
FWIW I find it easier to read if the type is explicit when it's not obvious from the RHS (there's a few cases like this)

Reply:
I did this to avoid duplicating the constant in `SmallString<128>`. What's preferable then?

Reply:
My other comment, ie. return `std::string` instead IMO

Reply:
Will go the `std::string` route

Reply:
Updated to std::string